src/logic/Windows.swift

```diff
@@ -162,9 +162,20 @@ class Windows {
     /// see: https://github.com/lwouis/alt-tab-macos/issues/1540
     static func detectTabbedWindows() {
         let cgsWindowIds = Spaces.windowsInSpaces(Spaces.idsAndIndexes.map { $0.0 })
+        let visibleCgsWindowIds = Spaces.windowsInSpaces(Spaces.idsAndIndexes.map { $0.0 }, false)
         list.forEach {
             if let cgWindowId = $0.cgWindowId {
-                $0.isTabbed = !cgsWindowIds.contains(cgWindowId)
+                if $0.isMinimized || $0.isHidden {
+                    if #available(macOS 13, *) {
+                        // not exact after window merging
+                        $0.isTabbed = !cgsWindowIds.contains(cgWindowId)
+                    } else {
+                        // not known
+                        $0.isTabbed = false
+                    }
+                } else {
+                    $0.isTabbed = !visibleCgsWindowIds.contains(cgWindowId)
```

Comment:
Maybe here we could do this to avoid making the call if not needed?

```swift
if let visibleCgsWindowIds = visibleCgsWindowIds {
  $0.isTabbed = !visibleCgsWindowIds.contains(cgWindowId)
else {
  visibleCgsWindowIds = Spaces.windowsInSpaces(Spaces.idsAndIndexes.map { $0.0 }, false)
}
```